oauthlib/oauth2/rfc6749/grant_types/base.py

```diff
@@ -116,14 +116,32 @@ def register_code_modifier(self, modifier):
     def register_token_modifier(self, modifier):
         self._token_modifiers.append(modifier)
 
-
     def create_authorization_response(self, request, token_handler):
+        """
+        :param request: OAuthlib request.
+        :type request: oauthlib.common.Request
+        :param token_handler: A token handler instace, for example of type
+                              oauthlib.oauth2.BearerToken.
+        """
         raise NotImplementedError('Subclasses must implement this method.')
 
     def create_token_response(self, request, token_handler):
+        """
+        :param request: OAuthlib request.
+        :type request: oauthlib.common.Request
+        :param token_handler: A token handler instace, for example of type
```

Comment:
Typo error `instace` i/o `instance` in all token_handler locations :-)

Reply:
ack! sorry! dealing with the merge-commit messages was so messy, so I just redid all the work.